pkg/cmd/cli/plugin/remove.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 the Velero contributors.
+Copyright 2017, 2021 the Velero contributors.
```

Comment:
Now, we're using the format without year. [1]
```suggestion
Copyright the Velero contributors.
```

Reply:
chnaged to `Copyright the Velero contributors.`